### PR TITLE
Release 1.19

### PR DESCRIPTION
**Pull request by nckturner**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Bump manifest version to v1.19.1-alpha.1

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nckturner** [comment]
Will merge this into master and then merge master into release-1.19

**Review by M00nF1sh — APPROVED** [review]
/lgtm
/approve

File: manifests/aws-cloud-controller-manager-daemonset.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
       serviceAccountName: cloud-controller-manager
       containers:
         - name: aws-cloud-controller-manager
-          image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.18.0-alpha.1
+          image: gcr.io/k8s-staging-provider-aws/cloud-controller-manager:v1.19.1-alpha.1
```
Comment (andrewsykim) [inline manifests/aws-cloud-controller-manager-daemonset.yaml:31]: I don't think we require patch versions to match, so this should be v1.19.0-alpha.1?

Comment (andrewsykim) [reply manifests/aws-cloud-controller-manager-daemonset.yaml:31]: Also, how did you build this without https://github.com/kubernetes/cloud-provider-aws/pull/136, or we didn't build this yet?

Comment (andrewsykim) [reply manifests/aws-cloud-controller-manager-daemonset.yaml:31]: cc @nicolehanjing

**Pull request merged**